src/net/sourceforge/kolmafia/FamiliarData.java

```diff
@@ -199,6 +202,8 @@ public FamiliarData(
       final int id, final String name, final int weight, final AdventureResult item) {
     this.id = id;
     this.name = name;
+    this.owner = KoLCharacter.getUserName();
+    this.ownerId = KoLCharacter.getUserId();
```

Comment:
Minor, but worth calling `setOwner` here? The method already exists, it more clearly shows these two fields are related.

Reply:
OK.